pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -709,11 +708,13 @@ func verifyStatus(ctx *domainContext, status *types.DomainStatus) {
 				status.Key())
 			publishDomainStatus(ctx, status)
 		}
-		// check if qemu processes has crashed
-		if configActivate && status.Activated && domainStatus == types.BROKEN {
-			errStr := fmt.Sprintf("verifyStatus(%s) device model process crashed", status.Key())
+		// check if any supporting tasks (like qemu) have crashed
+		if configActivate && status.Activated && !(domainStatus == types.BOOTING ||
+			domainStatus == types.RUNNING ||
```

Comment:
Can't we see the new PAUS* states here as well as part of the create/unpause sequence? That shouldn't be reported as an error.

Reply:
Hey @eriknordmark did you look at the commit I linked? It appears you're still looking at some stale commits. The latest one has:

```if configActivate && status.Activated && (domainStatus == types.BROKEN || domainStatus == types.UNKNOWN) {```